### PR TITLE
Refactor: Use react-hotkeys-hook lib to handle shortcuts usability

### PR DESCRIPTION
**Pull request by igorrCarvalho**
Here are some of the changes:
Ctrl + R: Open API Modal
Space: Open code Modal
Double Click: Open node settings

### CODE REVIEW

File: src/frontend/src/pages/FlowPage/components/nodeToolbarComponent/index.tsx [hunk 1]
```diff
@@ -65,6 +66,92 @@ export default function NodeToolbarComponent({
...
+    }
+    setNoticeData({
+      title: `${data.id} docs is not available at the moment.`,
+    });
+  }
+
+  function handleDownloadWShortcut(e: KeyboardEvent) {
+    e.preventDefault();
+    downloadNode(flowComponent!);
+  }
```
Comment (ogabrielluiz) [inline src/frontend/src/pages/FlowPage/components/nodeToolbarComponent/index.tsx:143]: https://github.com/JohannesKlauss/react-hotkeys-hook/blob/68387e593f9cbb79b1d06153a67afb33cf8cf0de/src/useHotkeys.ts#L17C7-L17C22

From what we can see in on the code above, I don't think we need the `preventDefault` here, right? Have you tested it without?

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]
Hey, Igor! Thanks for the PR

File: src/frontend/src/pages/FlowPage/components/nodeToolbarComponent/index.tsx (hunk 1 above)
Comment (igorrCarvalho) [reply src/frontend/src/pages/FlowPage/components/nodeToolbarComponent/index.tsx:143]: After testing, it appears that without using the preventDefault method, certain key combinations fail to activate the shortcut

**Comment by anovazzi1** [comment]
@igorrCarvalho is this still relevant? if yes, please fix the merge conflicts

**Comment by igorrCarvalho** [comment]
> @igorrCarvalho is this still relevant? if yes, please fix the merge conflicts

These changes has been merged into shortcuts_settings branch. Ill close this pr

**Pull request closed without merging**